src/renderer/screens/settings/index.js

```diff
@@ -107,6 +107,8 @@ const Settings = ({ history, location, match }: Props) => {
 
       if (idx > -1 && idx !== activeTabIndex) {
         setActiveTabIndex(idx);
+      } else {
+        setActiveTabIndex(0);
```

Comment:
Since you're always calling `setActiveTabIndex` here, you could use a ternary 

```suggestion
      setActiveTabIndex(idx > -1 && idx !== activeTabIndex ? idx : 0);
```

Reply:
Nice, commited, thanks